api/validation_models.py

```
@@ -366,43 +319,31 @@ class EnhancedSuccessResponse(BaseModel):
+
+    anonymize_instead_of_delete: bool = Field(default=False, description="Anonymize data instead of deletion")
+
+    reason: Optional[constr(max_length=500)] = Field(None, description="Reason for request (max 500 chars)")
+
+    @validator("request_type")
     def validate_request_type(cls, v):
         """Validate GDPR request type"""
-        valid_types = ['export', 'delete', 'anonymize']
+        valid_types = ["export", "delete", "anonymize"]
         if v not in valid_types:
             raise ValueError(f'Request type must be one of: {", ".join(valid_types)}')
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)